src/main/java/edu/cursor/springsecurity/controller/AuthorController.java

```diff
@@ -0,0 +1,31 @@
+package edu.cursor.springsecurity.controller;
+
+import edu.cursor.springsecurity.models.Author;
+import edu.cursor.springsecurity.service.IAuthorService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@AllArgsConstructor
+public class AuthorController {
+
+    private IAuthorService IAuthorService;
+
+    @PostMapping("/admin/add_author")
+    public void saveAuthor(@RequestBody Author author) {
+        IAuthorService.saveAuthor(author);
+    }
+
+    @GetMapping("/user/find_author/{authorID}")
```

Comment:
No underscores in url. Also, why do you need /user/ here? 
No need to call the variable "authorID".  Just id